documentation/docs/test.md

```diff
@@ -0,0 +1,55 @@
+# Set up PMM Client
```

Comment:
Q: I could not understand what this file is for :) I sort of see the name, but still don't get the purpose. Placeholder?